.github/workflows/dependabot-automerge.yml

```diff
@@ -0,0 +1,13 @@
+name: "Dependabot Automerge"
+
+on:
+  workflow_run:
+    workflows:
+      - "Web SDK Build & Test"
```

Comment:
So this means that whenever we put a PR in, this will run, right?

Reply:
Yeah, but technically it's when our Build and Test workflow "completes". So it'll only run if a PR finishes running tests. If we want, we can eventually tie this into other tasks, but I'm trying to be consistent with other projects that run this job.